entrypoint.sh

```diff
@@ -27,7 +27,7 @@ if [ "$1" = "test" ]; then
 elif [ "$1" = "lint" ]; then
     # Install standard linting dependencies; YMMV
     pip --quiet install \
-        .[lint] flake8 "isort<5.0.0" flake8-print flake8-logging-format flake8-isort
+        .[lint] "isort<5.0.0" flake8 flake8-print flake8-logging-format flake8-isort
```

Comment:
I re-introduced the isort pin after reverting this commit as still needed - no change here just ordering